_includes/news.html

```diff
@@ -0,0 +1,28 @@
+
+<section class="page-section text-secondary" id="recent-news">
+    <!-- ############################# News/achievement Section ##################################-->
+
+    <div class="container mb-5">
+        <div class="container">
+            <h2 class="page-section-heading text-center text-uppercase text-secondary mb-0">RECENT NEWS</h2>
+            <!-- Icon Divider -->
+            <div class="divider-custom divider">
+                <div class="divider-custom-line"></div>
+                <div class="divider-custom-icon">
+                    <i class="fas fa-newspaper"></i>
+                </div>
+                <div class="divider-custom-line"></div>
+            </div>
+            <ul>
+                {% for item in site.data.news %}
+                <li>
+                    <p>
+                        <b>[{{ item.date }}]</b> {{ item.description }}
+                    </p>
+                </li>
+                {% endfor %}
+
```

Comment:
Add a code only to show the top recent 10 news. Others go in **more news** hyperlinks 
When I click on **more news**, a new page should pop up which displays all the older news.
 
The pop-up page could be similar to the page opened when clicking any item on the `Undergraduate Project section`. 
<img width="1086" alt="image" src="https://github.com/user-attachments/assets/e08db96b-81ed-4101-b308-b107c959590d">